contracts/contracts/token/OUSD.sol

```diff
@@ -208,13 +224,13 @@ contract OUSD is Initializable, InitializableERC20Detailed, Governable {
         address _from,
         address _to,
         uint256 _value
-    ) public override returns (bool) {
+    ) external returns (bool) {
         require(_to != address(0), "Transfer to zero address");
-        require(_value <= balanceOf(_from), "Transfer greater than balance");
+        require(_value <= _allowances[_from][msg.sender], "Allowance exceeded");
 
-        _allowances[_from][msg.sender] = _allowances[_from][msg.sender].sub(
-            _value
-        );
+        _allowances[_from][msg.sender] =
```

Comment:
⚪️ This can be `-=`

Reply:
agreed: https://github.com/OriginProtocol/origin-dollar/pull/2298/commits/e949129d0669caebfdbaa72109c34adc0c888732